hal/armv7a/arch/cpu.h

```diff
@@ -49,9 +49,15 @@
 #define SYSTICK_INTERVAL 1000
 
 
+#define STACK_ALIGN 8
+
+
+#define STACK_SIZE(t) ((sizeof(t) + 3u) & ~0x3u)
```

Comment:
Name is kinda misleading (suggest size of the whole stack, not it's element), in kernel we have a convention that attribute is first in the name (see SIZE_KSTACK, SIZE_PAGE etc)
```suggestion
#define SIZE_STACK_ARG(t) ((sizeof(t) + 3u) & ~0x3u)
```